projects/firefox/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://www.mozilla.org/firefox/"
+primary_contact: "agaynor@mozilla.com"
+fuzzing_engines:
+- none
```

Comment:
I don't think this is right, probably should be libFuzzer?

Reply:
True.